tracer/src/Datadog.Trace/Iast/TruncationUtils.cs

```diff
@@ -33,4 +33,22 @@ public static void WriteTruncatableValue(this JsonWriter writer, string? value,
             writer.WriteValue(value);
         }
     }
+
+    public static void InsertTruncableValue(Dictionary<string, object> dict, string key, string? value, int maxValueLength)
+    {
+        if (value is null)
+        {
+            return;
+        }
+
+        if (value.Length > maxValueLength && maxValueLength > 0)
+        {
+            dict[key] = value.Substring(0, maxValueLength);
+            dict[TRUNCATED] = RIGHT;
```

Comment:
If multiple values are truncated, do you need to track which one? Right now there can only be a single `truncated:right` value in the dictionary.

Reply:
Each source is associated to a value, and when that value is truncated, "truncated:right" value will be set, so we can know which source have been truncated